example-get-started-experiments/README.md

```diff
@@ -24,20 +24,14 @@ git tag --sort=creatordate | xargs -n 1 git push --force frozen
 
 Run these to drop and then rewrite the experiment references on the repo:
 
-```
+```shell
 source .venv/bin/activate
 dvc exp remove -A -g origin
 dvc exp remove -A -g fixture
 dvc exp push origin -A
 dvc exp push fixture -A
 ```
 
-And this to clean the remote cache to only contain the last iteration:
```

Comment:
Whi dropping this?

Reply:
it's dangerous - we use the same remote across multiple repos (e.g. fixture and non-fixture, etc).